lib/commentOnOpenedPullRequest.js

```diff
@@ -32,6 +32,19 @@ function commentOnOpenedPullRequest(body, repositorySettings) {
     return commentOnOpenedPullRequest._implementation(filesUrl, commentsUrl, repositorySettings, userName, repositoryUrl, baseBranch, headBranch);
 }
 
+/**
+ * Transform a GitHub API URL to a regular GitHub URL.
+ *
+ * @param {String} apiURL The GitHub API URL.
```

Comment:
I think with camel case, this should be `apiUrl`. Same comment for `finalUrl`, and `getUrlFromApiUrl`.